### PR TITLE
iOS SDK rewrite

### PR DESCRIPTION
**Pull request by lukasgoetzweiss**

### CODE REVIEW

File: docs/sdks/client-sdks/ios.md [hunk 1]
```diff
@@ -13,23 +15,49 @@ While in XCode:
...
+    subjectAttributes: user.attributes,
+    defaultValue: "control"
+);
 ```
 
-During initialization, the SDK sends an API request to Eppo to retrieve the most recent experiment configurations such as variation values and traffic allocation. The SDK stores these configurations in memory so that assignments are effectively instant. For more information, see the [architecture overview](/sdks/architecture) page.
+During initialization, the SDK sends an API request to Eppo to retrieve the most recent experiment configurations: variation values, traffic allocation, etc. The SDK stores these configurations in memory, meaning assignments are effectively instant (accordingly, assignment is a synchronous operation). For more information, see the [architecture overview](/sdks/architecture) page.
 
-If you are using the SDK for experiment assignments, make sure to pass in an assignment logging callback (see [section](#define-an-assignment-logger-experiment-assignment-only) below).
+Eppo's SDK also supports providing the configuration directly at initialization. For more information, see the [initialization modes](#initialization-modes) section below.
```
Comment (aarsilv) [inline docs/sdks/client-sdks/ios.md:45]: 👍

File: docs/sdks/client-sdks/ios.md [hunk 2]
```diff
@@ -44,90 +72,220 @@ func segmentAssignmentLogger(assignment: Assignment) {
...
+try EppoClient.initializeOffline(
+    sdkKey: "SDK-KEY-FROM-DASHBOARD",
+    initialConfiguration: try Configuration(
+        flagsConfigurationJson: Data(#"{ "pre-loaded-JSON": "passed in here" }"#.utf8),
+        obfuscated: false
+    )
+);
+```
+
+The `obfuscated` parameter is used to inform the SDK if the flag configuration is obfuscated.
```
Comment (aarsilv) [inline docs/sdks/client-sdks/ios.md:237]: 👍

**Review by aarsilv — APPROVED** [review]
I don't know Swift, but everything you wrote checks out with me!

File: docs/sdks/client-sdks/ios.md [hunk 3]
```diff
@@ -44,90 +72,220 @@ func segmentAssignmentLogger(assignment: Assignment) {
...
+More details about logging and examples (with Segment, Rudderstack, mParticle, and Snowplow) can be found in the [event logging](/sdks/event-logging/) page.
+:::
+
+## Appendix
+
+### Publishing releases
+
+Swift Package Manager relies on semantic versioning without a prefix, such as `v`.
+
+When publishing a release of the Swift SDK, use a git tag such as `3.2.1`.
```
Comment (rasendubi) [inline docs/sdks/client-sdks/ios.md:291]: This is a developer-only documentation. Users don't need to know this
```suggestion
```

**Review by rasendubi — APPROVED** [review]
Almost got a heart attack when received a notification with just the PR title. But otherwise looks good to me

File: docs/sdks/client-sdks/ios.md (hunk 3 above)
Comment (lukasgoetzweiss) [reply docs/sdks/client-sdks/ios.md]: Hahah yeah sorry should have included "docs" in the title 😅

**Pull request merged**